rdmo/core/settings.py

```diff
@@ -327,6 +327,26 @@
 
 OPTIONSET_PROVIDERS = []
 
+PROJECT_VALUES_VALIDATION = True
```

Comment:
as discussed, please make the validation default to `False` (so that they are disabled)